packages/manager/modules/pci/src/projects/project/kubernetes/add/add.controller.js

```diff
@@ -124,9 +137,43 @@ export default class {
       quota: find(this.quotas, { region: this.cluster.region.name }),
     });
     this.loadFlavors(this.cluster.region.name);
+    this.loadPrivateNetworks();
     this.displaySelectedRegion = true;
   }
 
+  loadPrivateNetworks() {
+    this.availablePrivateNetworks = [
+      this.defaultPrivateNetwork,
+      ...sortBy(
+        map(
+          filter(this.privateNetworks, (network) => {
+            const region = find(network.regions, {
+              region: this.cluster.region.name,
+              status: 'ACTIVE',
+            });
+            set(network, 'clusterRegion', region);
+            return region;
+          }),
+          (privateNetwork) => ({
+            ...privateNetwork,
+            name: `${privateNetwork.vlanId.toString().padStart(4, '0')} - ${
+              privateNetwork.name
+            }`,
+          }),
+        ),
+        ['name'],
+      ),
+    ];
+    if (
+      this.cluster.privateNetwork &&
+      filter(this.availablePrivateNetworks, {
```

Comment:
`find` might fit in better in this place I guess :)

Reply:
👍